setup.py

```diff
@@ -21,9 +54,8 @@
         'Programming Language :: Python',
         'Framework :: Django',
     ],
-    install_requires=[
-        'shortuuid>=0.4.3,<1.0.0'
-    ],
+    install_requires=load_requirements('requirements/requirements.txt', 'requirements/requirements-django.txt'),
```

Comment:
I've never been a fan of loading requirements from the `requirements.txt` file for the simple fact that it is not a standard, and requires a custom function like the one you've made. Any particular reason this concept was introduced? Is that something we are doing in all of our repos now?